content/docs/atom-guide/look-dev/tools/material-editor.md

```diff
@@ -0,0 +1,90 @@
+---
+title: Material Editor
+description: Learn how to use the Material Editor to create materials in Atom.
+toc: true
+---
+
+## Overview
+The **Material Editor** is a standalone application that allows artists to view, edit, and create new materials. This section introduces the Material Editor and describes the editor layout and its individual panels.
+
+For more information about features common to Atom tools, menus, dockable panels, and working with documents, see [Atom Tools Common Features](/docs/atom-guide/look-dev/tools/atom-tools-common-features/). 
+
+## Quick Start
+To get started, launch the Material Editor and create or edit a material. 
+
+### Launching Material Editor
+There are multiple ways to launch Material Editor.
+- From **Open 3D Engine (O3DE)** Editor select **Main Menu > Tools > Material Editor**
+- From the Material Component context menu, select **Open Material Editor...**
+- From the Asset Browser, double click on a ".material" or ".materialtype".
+- From the Asset Browser, right click on a ".material" or ".materialtype", then select **Open in Material Editor...** 
+- Material Editor is also a standalone executable that can be launched directly from a file browser or command console.
+  - Launch the executable `<build>\bin\profile\MaterialEditor.exe`
+  - This requires passing the �project-path, followed by the path your project, as a command line argument.
+
+### Creating Materials
+You have multiple options to start editing a material.
+- Create a new material from a material type.
+  - Select **Main Menu > File > New...**.
+    - This opens the "Create Material Document" dialog.
+  - Select a material type as the basis for your new material.
+  - Select a path and file name for the new material.
+- Open an existing material
+  - Select **Main Menu > File > Open...**.
+- Open or create a material from the Asset Browser.
+
+### Editing Materials
+- All material editing is done by changing properties in the Material Editor inspector.
+- Changes are recorded as properties are edited and can be undone and redone. 
+
+### Child Materials
+- Materials can reference a parent material to inherit and override its properties.
+- Changes made to the parent, or another ancestor, are automatically reflected in the child material unless the child is overriding those properties.
+- Child materials can be created opening an existing material and selecting **Main Menu > File > Save As Child...**. 
+
+### Mirroring Changes in The O3DE Editor
+- Materials saved in the Material Editor will be processed by the Asset Processor.
+- If the O3DE Editor is open and the saved material is assigned to a Material Component or terrain it will automatically not reload to reflect the changes.
+- Enabling the autosave in Material Editor settings dialog will causes updates to be continuously mirrored in the O3DE Editor.
```

Comment:
Correct. As long as the AP is running, if you make changes in the material editor and save them then the main editor will be notified and display those changes. Previously, this was not working correctly and did not work without switching focus back to the main editor. It is now possible to have both of them open simultaneously, make changes in the material editor, save those changes, and see things update in the main editor without switching focus. Additionally, there is now an automatic save feature in the material editor and other tools. Enabling the automatic save feature in material editor will cause any of changes to automatically be mirrored in close to real time in the main editor. The same is true of material canvas except that material canvas does not need auto saving enabled because it's generating the files whenever you make changes. Given time, I would promote this workflow and deprecate the material instance editor.